app/src/main/res/values/strings-preferences.xml

```diff
@@ -73,7 +73,7 @@
 	<string name="prefs_displaytrack_osm_summary_ask">Always display OpenStreetMap background? Requires a data connection</string>
 
 	<string name="prefs_output">GPX settings</string>
-	<string name="prefs_storage_dir">External storage (SD) directory</string>
+	<string name="prefs_storage_dir">Public directory (in documents)</string>
```

Comment:
Why not: "Storage folder in documents" ?

Because this setting allow user to set the name of the subfolder inside document folder where the GPX will be stored.

Reply:
That sounds good! I agree with the suggestion of using "Storage folder in documents." It accurately reflects that the setting allows the user to define the subfolder within the Documents directory where the GPX files will be stored.